experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts

```diff
@@ -403,8 +408,8 @@ export class SharedPropertyTree extends SharedObject {
 				unrebasedRemoteChanges: this.unrebasedRemoteChanges,
 			};
 			const chunkSize = 5000 * 1024; // Default limit seems to be 5MB
-			let serializedSummary =
-				serializer !== undefined ? serializer.stringify(summary, this.handle) : JSON.stringify(summary);
+			let serializedSummary = JSON.stringify(summary);
+				// serializer !== undefined ? serializer.stringify(summary, this.handle) : JSON.stringify(summary);
```

Comment:
@ruiterr - Just FYI - In the past I've wondered if we should encode IFluidHandles on write and decode them on read so we can use vanilla JSON.stringify()/parse().

Reply:
With the PropertyDDS, we would have the option to introduce a special type for Handles. In that case, we could use the implementation for this type to decode them and wouldn't have to do it everywhere within the JSON.

I just realized, that I didn't include my changes to use the binary serialization. I have to check where I have those.